sync-superchain.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# This script is used to sync superchain configs in the registry with OP Geth.
+
+set -euo pipefail
+
+# Constants
+REGISTRY_COMMIT="9596f616b9bd07b7823e23aac72ae4621805653d"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+repodir=$(mktemp -d)
+workdir=$(mktemp -d)
+
+# Clone the registry
+echo "Cloning SR..."
+cd "$repodir"
+git clone https://github.com/ethereum-optimism/superchain-registry.git
+cd "$repodir/superchain-registry"
+git checkout "$REGISTRY_COMMIT"
```

Comment:
```suggestion
git clone --no-checkout --depth 1 --shallow-submodules https://github.com/ethereum-optimism/superchain-registry.git
cd "$repodir/superchain-registry"
git fetch --depth 1 origin "$REGISTRY_COMMIT"
git checkout "$REGISTRY_COMMIT"
```
may greatly speed up the clone, since we don't need any other revision than the one specified above.

Unfortunately `git clone` doesn't allow us to directly specify the commit hash, so we have to do a shallow clone, followed by a shallow fetch...

Reply:
nice great idea